panels/tests/test_event.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
These tests were moved here from the core repo.  I didn't write any new tests.  Someday in the far future we won't be so busy implementing new functionality that we'll actually have time to write tests.  Someday.